test/test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+export PYTHONPATH="$(realpath $(dirname $0))":"$(realpath $(dirname $0)/..)":"$PYTHONPATH"
+
+cd "$(realpath $(dirname $0))"
+PYTHON=python2
+if  head -1 ../init | grep -q python3; then
+	PYTHON=python3
+fi
+$PYTHON test_errors.py -v
```

Comment:
test_errors.py doesn't exist on master yet, is there a PR/branch to add it and the supporting files?